pwem/protocols/protocol_export/protocol_export_DB.py

```diff
@@ -181,11 +198,19 @@ def exportFSCStep(self):
     def exportMasksStep(self):
         outputDir = os.path.join(self.dirName, self.MASKDIR)
         self.createDirectoryStep(outputDir)
-        ih = ImageHandler()
+
         for counter, mask in enumerate(self.exportMasks, 1):
-            ih.convert(mask.get().getLocation(),
-                       os.path.join(outputDir,
-                                    self.MASKNAME % counter))
+            mask = mask.get()
+            inVolFileName = mask.getFileName()
+            outVolFileName = os.path.join(outputDir,
+                                    self.MASKNAME % counter)
+            shifts = mask.getOrigin(force=True).getShifts()
+            sampling = mask.getSamplingRate()
+
+            ccp4header = Ccp4Header(inVolFileName)
```

Comment:
Are we sure at this point that all masks are mrc? Otherwise we can not use Ccp4Header, or we must convert the mask before this.

Reply:
Good point. I do not think we can.  Let me think about it.

    Roberto

Reply:
ready to merge, then?